packages/beacon-node/src/metrics/validatorMonitor.ts

```diff
@@ -115,12 +120,12 @@ type ValidatorStatus = {
   inclusionDistance: number;
 };
 
-function statusToSummary(status: AttesterStatus): ValidatorStatus {
-  const flags = parseAttesterFlags(status.flags);
+function statusToSummary(inclusionDelay: number, flag: number, isActiveInCurrentEpoch: boolean): ValidatorStatus {
+  const flags = parseAttesterFlags(flag);
   return {
     isSlashed: flags.unslashed,
-    isActiveInCurrentEpoch: status.active,
-    isActiveInPreviousEpoch: status.active,
+    isActiveInCurrentEpoch,
+    isActiveInPreviousEpoch: isActiveInCurrentEpoch,
```

Comment:
the change matches old implementation, however looks like a bug because `status.active` is applied for both `isActiveInCurrentEpoch, isActiveInPreviousEpoch`

we can just pass in `isActiveInPrevEpoch` here from `beforeProcessEpoch()`